community-containers/npmplus/readme.md

```diff
@@ -0,0 +1,17 @@
+## NPMplus
+This container contains a fork of the Nginx Proxy Manager, which is a WebUI for nginx. It will also automatically create a config and cert for AIO.
```

Comment:
I guess we should add a hint also to the caddy community container that you cannot use both at the same time?

Reply:
yes